### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by jstiers** [jrnewell/spotify-ripper#218]
# Hey Guys
## Could you explain why I'm getting this error?
I seem to be too stupid.

Thank you :)
```
MacBook-Pro:~ $ spotify-ripper h
Traceback (most recent call last):
  File "/Users/.pyenv/versions/2.7.10/bin/spotify-ripper", line 9, in <module>
    load_entry_point('spotify-ripper==2.8.9', 'console_scripts', 'spotify-ripper')()
  File "/Users/.pyenv/versions/2.7.10/lib/python2.7/site-packages/pkg_resources/__init__.py", line 552, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/Users/.pyenv/versions/2.7.10/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2672, in load_entry_point
    return ep.load()
  File "/Users/.pyenv/versions/2.7.10/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2345, in load
    return self.resolve()
  File "/Users/.pyenv/versions/2.7.10/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2351, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/Users/.pyenv/versions/2.7.10/bin/main.py", line 7, in <module>
    from spotify_ripper.ripper import Ripper
  File "/Users/.pyenv/versions/2.7.10/lib/python2.7/site-packages/spotify_ripper/ripper.py", line 9, in <module>
    from spotify_ripper.progress import Progress
  File "/Users/.pyenv/versions/2.7.10/lib/python2.7/site-packages/spotify_ripper/progress.py", line 11, in <module>
    import spotify
  File "/Users/.pyenv/versions/2.7.10/lib/python2.7/site-packages/spotify/__init__.py", line 81, in <module>
    from spotify._spotify import ffi, lib  # noqa
ImportError: dlopen(/Users/.pyenv/versions/2.7.10/lib/python2.7/site-packages/_cffi_backend.so, 2): Symbol not found: _PyUnicodeUCS2_AsASCIIString
  Referenced from: /Users/.pyenv/versions/2.7.10/lib/python2.7/site-packages/_cffi_backend.so
  Expected in: flat namespace
 in /Users/.pyenv/versions/2.7.10/lib/python2.7/site-packages/_cffi_backend.so
```

**Comment by hover20** [comment]
Hello, 

I already opened a Issue #214 with the same error message. I think I found a solution - to proof. See #214 .

**Comment by jrnewell** [comment]
Can I close this, or is this still an issue?

**Comment by hover20** [comment]
Yes, I think the problem is solved by #214
